### PR TITLE
drop remaining netaddr

### PR DESCRIPTION
**Pull request by michalskrivanek**
commit fd20e99066c34006f392409b96150b29d0968470 dropped python-netaddr
dependency, remove remaining usage of filters requiring it

### CODE REVIEW

**Review by mwperina — APPROVED** [review]
+1

**Comment by michalskrivanek** [comment]
fix works. however test_upload_cirros_image is screwed up both on el8 and el9 but I believe that's a different issue

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -49,7 +52,7 @@ jobs:
             "ansible-lint>=6.0.0,<7.0.0" \
             cryptography \
             "pyyaml>=5.4.1"
-          ansible-lint roles/* -x 204
+          ansible-lint -x 'key-order[task],no-free-form,schema[vars],yaml[octal-values],risky-shell-pipe,fqcn[keyword]' roles/*roles/*
```
Comment (mnecas) [inline .github/workflows/build.yaml:55]: Think you have typo in `roles/*roles/*` should be `roles/*`

**Review by mnecas — CHANGES_REQUESTED** [review]

**Pull request merged**